docs/section23/Rule23-1.md

```diff
@@ -0,0 +1,34 @@
+
+# Airside System - Rule 23-1 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** False   
+**Rule ID:** 23-1  
+**Rule Description:** System 2 and 4 - Electric air-source heat pumps shall be modeled with electric auxiliary heat and an outdoor air thermostat. The systems shall be controlled to energize auxiliary heat only when the outdoor air temperature is less than 40°F. The air-source heat pump shall be modeled to continue to operate while auxiliary heat is energized.  
```

Comment:
I wonder why we decided to make these rule non-primary. It will trigger an undertermined outcome for all projects with baseline systems 2 & 4 (e.g., all residential projects in cooling-dominated climates). Plus, it seems that there is an easy way to verify it by checking HeatingSystem.h e a t p u m p _ a u x i l l i a r y _ h e a t _ t y p e and h e a t p u m p _ l o w _ s h u t o f f _ t e m p e r a t u r e. The only missing piece in the schema is for verifying the part that "The air-source heat pump shall be modeled to continue to operate while auxiliary heat is energized." Perhaps add HeatingSystem.heatpump_operate_when_auxilliary_heat (true/false).

Reply:
@mkarpman - I think you're right that we can easily change this to a primary rule.

For the requirement for heat pumps to operate while the auxiliary heat is also running, there is `heatpump_low_shutoff_temperature` - can we require that this temperature be lower than a certain number?  Perhaps a lookup table based on climate zone?

Reply:
Do we know what the heatpump_low_shutoff_temperature should be? What would we be checking for in both Karen and Maria's suggestion I am not sure what the correct value would be? Does anyone have an answer?

Reply:
the `heatpump_auxilliary_heat_high_shutoff_temperature` should be 40F.

suggestion is to check that `heatpump_low_shutoff_temperature` is less than `heatpump_auxilliary_heat_high_shutoff_temperature`, results in PASS

Reply:
update based on Maria's suggested logic

![image](https://github.com/pnnl/ruleset-checking-tool/assets/114143532/b0191ab0-95d2-46e1-b558-3b2b90dad26d)